modules/ROOT/pages/deploy-to-rtf.adoc

```diff
@@ -264,7 +255,10 @@ include::mule-runtime::partial$mmp-concept.adoc[tag=encryptCredentials]
 ----
 +
 NOTE: Make sure that the username and password are not set in the deployment configuration, or they will overwrite the defined server ID.
+=======
+include::mule-runtime::partial$mmp-deploy-to-rtf.adoc[tag=rtfEncryptCredentials]
+>>>>>>> v4.3
 
 == See Also
 
-* xref:mmp-concept.adoc[Mule Maven Plugin]
+* xref:mule-runtime::mmp-concept.adoc[Mule Maven Plugin]
```

Comment:
TODO: why is `mule-runtime::` needed here?

Reply:
@Cristian-Venticinque

Reply:
Yes, adding the component instructs the include directive to always point to **latest**.  This is useful in this case because at the moment, all MMP includes go to v4.3, but once we release v4.4, the includes would look for the content in that repo without having to update anything. The only thing we need to do is to copy the existing partials from v4.3 to v4.4